### DISCUSSION TITLE
Let the redstone device know that the chunk is about to be uninstalled

### DISCUSSION THREAD

**Discussion by ApliNi** [PurpurMC/Purpur#993] [__Dismissed__] [answered]
Specifies a block that emits a redstone signal when there are no players within a certain distance around it or when the chunk is about to be unloaded, This block will act as a switch for redstone devices. Cooperate with the Chunk uninstall time of paper, this may protect some redstone devices that cannot be uninstalled directly, when the player is disconnected!

**Comment by katekerllenevich** [comment, marked as answer]
This is far out of scope for Purpur, a plugin is the best way to do this.
